### ISSUE TITLE
More expresive views

### ISSUE THREAD

**Issue by happyDemon** [modulargaming/modulargaming#101]
I think it might be helpful for templating in general if we'd add every available property in their respective views (meaning every property we would set in the controller when filling the view class with data).

This would make it easier to reference them from a templating stand point and wouldn't require a developer to go peek at the controller code.

For certain properties we could add some docblock comments to make things clearer, making it easier for outside developers to jump in (thus lowering the learning curve).
- [ ] Admin
- [ ] Core
- [ ] Forum
- [ ] Game
- [ ] Item
- [ ] Message
- [ ] Pet
- [ ] User

**Comment by Hinton** [comment]
Yes, definitely a good idea. I've been trying to do so myself whenever I use a property. We should use comments to describe the type of the property.

**Comment by happyDemon** [comment]
I've been putting it off xD _naughty me_ o.O
But it's something that I've been starting to worry about since we're 2 months away from a release.

**Comment by curtisdelicata** [comment]
Should this include passing data as_array instead of objects?

**Comment by Hinton** [comment]
@curtisdelicata We should use the same design everywhere, and as we need to use lazy relationships in some places it is better to send the objects.
